crates/bevy_xpbd_3d/examples/revolute_joint_3d.rs

```diff
@@ -56,10 +56,10 @@ fn setup(
 
     commands.spawn(
         RevoluteJoint::new_with_compliance(anchor, object, 0.0)
-            .with_local_anchor_1(Vec3::Y * -0.5)
-            .with_local_anchor_2(Vec3::Y * 0.5)
-            .with_aligned_axis(Vec3::Z)
-            .with_angle_limits(-0.1 * std::f32::consts::PI, 0.5 * std::f32::consts::PI),
+            .with_local_anchor_1(Vector::Y * -0.5)
```

Comment:
I've been thinking a bit... I'm worried that the examples become worse examples by generic like this. It's not how people would actually use the library. They'll likely just choose one precision level, `f32` or `f64` and then use those types instead of `Scalar`, `Vector` etc.. i.e. I think the library looks less ergonomic than it actually is...

Maybe it's worth duplicating at least some of the examples? They could just be feature gated.

Reply:
Yeah, I agree. It's better to have the examples be as simple as possible and reflect actual usage.

I'll work on this later today when I fix the errors as well (insta snapshots changed due to f64 type I think)

Reply:
Done, I changed all the examples to use `f32` again and added it as a required feature for the examples. I didn't add any `f64` examples yet, but we can add them later if we want.

Reply:
Okay I added `cubes_f64` and it seems to work as expected.